giotto/mapper/utils/visualization.py

```diff
@@ -10,14 +12,13 @@ def _get_node_size(node_elements):
 
 def _get_node_text(graph):
     return [
-        'Node Id:{}<br>Node size:{}<br>Pullback Id:{}<br>Cluster label:{}'
+        'Node Id:{}<br>Node size:{}<br>Cluster label:{}'
```

Comment:
Is 'Node Id' a good name? The `for interval_id` makes me think this is the id of the interval in the filter range.

Reply:
'Cluster label' is, AFAIK, a relative label within the pullback cover set in question. Discussion: is the name sufficiently descriptive? Or do we need to be even more explicit, e.g. with 'Relative cluster label' or sth? I really am not sure.

Reply:
Both good points. Since it seems reasonable to assume most of our users will have some basic knowledge of Mapper, why not adopt the following:

* Pullback cover set Id
* Partial cluster label

I propose "partial" because that's how Carlsson et al refer to clustering in their paper.

Reply:
Approved!